openmp/runtime/src/kmp_csupport.cpp

```diff
@@ -4515,6 +4515,15 @@ void omp_free(void *ptr, omp_allocator_handle_t allocator) {
 }
 /* end of OpenMP 5.1 Memory Management routines */
 
+void *omp_get_dyn_groupprivate_ptr(size_t offset, int *is_fallback,
+                                   omp_access_t access_group) {
+  if (is_fallback != NULL)
```

Comment:
```suggestion
  if (is_fallback != nullptr)
```

Is there a reason to have an optional out argument? Why not just make it unconditional or return a pair?

Reply:
This interface was discussed in the OpenMP language committe, and it's the version that was accepted. Having an optional out argument, we can pass nullptr, or no parameter in the C++ and Fortran versions.